debian/snapd.install

```diff
@@ -1,4 +1,5 @@
 /usr/bin/snap
+/usr/bin/snapctl
```

Comment:
/usr/lib/snapd sounds a bit nicer, as it means hiding something that unusable for most people, and also having tab-completion on bare "snap" completing the space.